.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,54 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '34 10 * * 2'
```

Comment:
Is there a need for anything other than a push to `main`?

Reply:
It is not necessary, it's just two different trigger types. Even if there was no updates on the main branch, the Scorecard Action would still weekly search for vulnerabilities.

I've removed the schedule trigger but if you prefer to keep it let me know that I'll add it again